src/config/local.ts

```diff
@@ -19,9 +19,11 @@ const config: ConfigInterface = {
     test: false,
     forceLocal: false,
   },
-  telemetry: {
-    posthogKey: 'phc_xvAQWfpHug7G0SuU5P9wwAbvP9ZawgAfIEZ9FUsiarS',
-  },
 };
+if (process.env.IASQL_TELEMETRY === 'on') {
+  config.telemetry = {
+    posthogKey: 'phc_xvAQWfpHug7G0SuU5P9wwAbvP9ZawgAfIEZ9FUsiarS',
```

Comment:
there is no sentry in the run/server part?

Reply:
Nope. However in Next.js land I have no idea if the setup for Sentry applies to both or not. The `server-config` section is meant for stuff that absolutely shouldn't live in the front-end.